src/Core/Block.cpp

```diff
@@ -624,6 +625,18 @@ Names Block::getDataTypeNames() const
 }
 
 
+std::unordered_map<String, size_t> Block::getNamesToIndexesMap() const
```

Comment:
Can't we just return `index_by_name` here? Also can make return type `IndexByName` (but now it's defined `private` section)